saleor/graphql/core/scalars.py

```diff
@@ -36,6 +36,24 @@ def parse_value(value):
             return None
 
 
+class PositiveDecimal(Decimal):
+    """Positive Decimal scalar implementation.
+
+    Should be used in places where value must be positive.
+    """
+
+    @staticmethod
+    def parse_value(value):
+        value = super(PositiveDecimal, PositiveDecimal).parse_value(value)
+        if not value:
+            return value
```

Comment:
This is super confusing at 1st glance, maybe a docstring would be helpfull?

Reply:
Still I don't quite get it. If there isn't value, return that nothing?
If so, this `if -> return` can be removed and lower `if` replaced with `if value and value < 0`